src/app/home/event-user-list/event-user-list.component.html

```diff
@@ -0,0 +1,25 @@
+<clr-modal #userListModal [(clrModalOpen)]="open" [clrModalSize]="'lg'">
```

Comment:
```suggestion
<clr-modal #userListModal [(clrModalOpen)]="open" clrModalSize="lg">
```

No need for dynamic binding here IMO